modules/oadp-1-4-0-release-notes.adoc

```diff
@@ -0,0 +1,64 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-06-28
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-0-release-notes_{context}"]
+= OADP 1.4.0 release notes
+
+The {oadp-first} 1.4.0 release notes lists resolved issues and known issues.
+
+[id="resolved-issues-1-4-0_{context}"]
+== Resolved issues
+
+.Restore works correctly in OpenShift Container Platform 4.16
+
+Previously, while restoring the deleted application namespace, the restore operation partially failed with the `resource name may not be empty` error in OpenShift Container Platform 4.16.
+With this update, restore works as expected in OpenShift Container Platform 4.16
+link:https://issues.redhat.com/browse/OADP-4075[OADP-4075]
+
+.Data Mover backups works properly in the OpenShift Container Platform 4.16 cluster
+
+Previously, Velero was using the lower version of SDK where the `Spec.SourceVolumeMode` field did not exist. As a consequence, Data Mover backups failed in the OpenShift Container Platform 4.16 cluster on the external snapshotter with v4.2 version. 
+With this update, external snapshotter is upgraded to v7.0 version and later. As a result, backups do not fail in the OpenShift Container Platform 4.16 cluster.
+link:https://issues.redhat.com/browse/OADP-3922[OADP-3922]
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+
+.Backup fails when checksumAlgorithm is not set for MCG
```

Comment:
This is better explained below in Changes from OADP 1.3 to 1.4, but should we be more verbose here as well, @weshayutin ?

Reply:
Added a link to point to the Changes from OADP 1.3 to 1.4 section.